src/test/java/seedu/guest/model/ModelManagerTest.java

```diff
@@ -36,15 +36,15 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
 
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
-        UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setGuestBookFilePath(Paths.get("address/book/file/path"));
-        userPrefs.setGuiSettings(new GuiSettings(1, 2, 3, 4));
-        modelManager.setUserPrefs(userPrefs);
-        assertEquals(userPrefs, modelManager.getUserPrefs());
+        GuestPrefs guestPrefs = new GuestPrefs();
+        guestPrefs.setGuestBookFilePath(Paths.get("address/book/file/path"));
```

Comment:
Should the file path be "guest/book/file/path" instead of "address/book/file/path"?